content/soda-gettingstarted/multicloud-ha-installation-using-ansible.md

```diff
@@ -0,0 +1,74 @@
+For installing SODA in default standalone configuration please refer to the [standalone installation guide](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/)
+
+## Pre-config for Deploying Multi-Cloud Services on Kubernetes Cluster (Ubuntu 16.04 or Ubuntu 18.04)
+All the installation work is tested on `Ubuntu 16.04` and `Ubuntu 18.04`, please make sure you have installed the right one. Also `root` user is REQUIRED before the installation work starts.
+The following configuations are required on the Kubernetes Cluster and the SODA installer node. 
+  1. A Kubernetes Cluster should be available for deploying the SODA Multi-Cloud Services on it.
+  2. The Kubernetes Cluster should have atleast three nodes to deploy the services. 
+  3. The Kubernetes Master node should have a user, (say 'k8s_user') configured to access the installer node.
+  4. [Passwordless SSH](https://help.ubuntu.com/community/SSH/OpenSSH/Keys) should be setup between the Kubernetes Master node and the SODA Installer node.
+
+
+All the other configurations remain same as the [standalone installation](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/). 
+Please follow the standalone installation guide for the remaining installation steps. 
+Here are the steps that are specific to the HA installation of Multi-Cloud Services on Kubernetes Cluster.
+
+
+### Configurations for Installing Multi-Cloud in HA mode.     
+The basic configurations for installing SODA Multi-Cloud in HA mode are the same as installing Multi-Cloud in the standalone mode. The few additional steps required to install Multi-Cloud in HA mode are 
+
+As a root user
+#### Download SODA installer code
+```bash
+git clone https://github.com/sodafoundation/installer.git
+cd installer/ansible
+# Checkout the required version. For example, to checkout v1.2.0 follow
+git checkout v1.2.0
+```
+{{% notice warning %}}
+Checkout the latest stable release. Current stable release: stable/faroe. If you want to get the master branch of all components, you can skip this step. (Master may not be stable or tested fully)
+{{% /notice %}}
+
+
+    - cd installer/ansible/group_vars
+    - Open the common.yml file
+      - Add the IP of the host (which is reachable from the Kubernetes node)
+      - export the HOST_IP=<IP-of-the-Host>
+      - Change 'gelato_ha: false' to 'gelato_ha: true' to install Multi-Cloud services in HA mode. 
+  
+    - Open the gelato-ha.yml file
+      - Add the user that has access to the Kubernete Cluster to deploy the services.
+      - The k8suser should be added to sudoers group.
+        - Eg. k8s_user: <k8suser> 
+  
+    - Open the local.hosts file in the <installer/ansible> dir.
+      - Add the IP and other details of the Kubernetes Master node under the group
+       - Eg. [k8smaster]
+             master ansible_port=22 ansible_user=root ansible_host=192.168.xx.yy       
```

Comment:
In the installer this line is commented. 
Mention that user has to un comment and then replace with appropriate values.

Reply:
done